project.go

```diff
@@ -0,0 +1,392 @@
+}
+
+func (f *commaSeparatedFlag) String() string {
+	return strings.Join(f.values, ",")
+}
+
+func (f *commaSeparatedFlag) Set(v string) error {
+	f.values = strings.Split(v, ",")
+	return nil
+}
+
+func (f *commaSeparatedFlag) validate(validValues []string) error {
+	var invalidValues []string
+	for _, cv := range f.values {
+		var found bool
+		for _, validValue := range validValues {
+			if validValue == cv {
+				found = true
+				break
+			}
+		}
+		if !found {
+			invalidValues = append(invalidValues, cv)
+		}
+	}
+	if len(invalidValues) > 0 {
+		return fmt.Errorf("invalid values: %s (valid options are: %s)", strings.Join(invalidValues, ", "), strings.Join(validValues, ", "))
+	}
+	return nil
+}
+
+type projectRemove struct {
+	cmd.ConfirmationCommand
+	name string
+	fs   *gnuflag.FlagSet
+}
+
+func (c *projectRemove) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-remove",
+		Desc: "removes the given project",
+	}
+}
+
+func (c *projectRemove) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	config, err := loadConfigFile()
+	if err != nil {
+		return errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	if !c.Confirm(ctx, fmt.Sprintf("Are you sure you want to remove the project %q?", c.name)) {
+		return nil
+	}
+	envNames := config.envNames()
+	appNames := make([]string, len(envNames))
+	for i, envName := range envNames {
+		appNames[i] = fmt.Sprintf("%s-%s", c.name, envName)
+	}
+	errs, err := deleteApps(appNames, client)
+	if err != nil {
+		return err
+	}
+	var notFound int
+	for _, err := range errs {
+		if err != nil {
+			if e, ok := err.(*tsuruerrors.HTTP); ok && e.Code == http.StatusNotFound {
+				notFound++
+			}
+		}
+	}
+	if notFound == len(errs) {
+		return errors.New("project not found")
+	}
+	fmt.Fprintln(ctx.Stdout, "Project successfully removed!")
+	return nil
+}
+
+func (c *projectRemove) Flags() *gnuflag.FlagSet {
+	if c.fs == nil {
+		c.fs = c.ConfirmationCommand.Flags()
+		c.fs.StringVar(&c.name, "name", "", "name of the project to b remove")
+		c.fs.StringVar(&c.name, "n", "", "name of the project to remove")
+	}
+	return c.fs
+}
+
+type projectInfo struct {
+	name string
+	fs   *gnuflag.FlagSet
+}
+
+func (c *projectInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-info",
+	}
+}
+
+func (c *projectInfo) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	apps, err := c.projectApps(client)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(ctx.Stdout, "Project name: %s\n\n", c.name)
+	var envs cmd.Table
+	envs.Headers = cmd.Row{"Environment", "Address", "Version", "Deploy date"}
+	for _, app := range apps {
+		row := cmd.Row{app.Env.Name, app.Addr, "", ""}
+		if deploy, err := lastDeploy(client, app.Name); err == nil && deploy.Image != "" {
+			row[2] = deploy.Image
```

Comment:
But isn't `version` how Tsuru calls it? I believe we should keep as consistent as possible with Tsuru nomenclature - `project` and `envs` were concepts we really needed (use) and Tsuru didn't offer. Despite this, I'd suggest we keep to default Tsuru naming. Therefore :+1:  to keeping `version`.